chore: rename `route` fixture in ct (#31817)
microsoft/playwright: docs/src/test-components-js.md

### Handling network requests

Playwright provides an **experimental** `router` fixture to intercept and handle network requests. There are two ways to use the `router` fixture:

> * Call `router.route(url, handler)` that behaves similarly to [`method: Page.route`]. See the [network mocking guide](./mock.md) for more details.

Reviewer: ```suggestion
* Call `router.route(url, handler)` that behaves the same as [`method: Page.route`]. See the [network mocking guide](./mock.md) for more details.
```

* Call `router.use(handlers)` and pass [MSW library](https://mswjs.io/) request handlers to it.

Here is an example of reusing your existing MSW handlers in the test.

```ts

> import { handlers } from '@src/mocks/handlers';

Reviewer: ```suggestion
import { handlers } from '../mocks/handlers';
```

maybe simpler like this?
